### PR TITLE
DOCS-902 - PGE Release Notes for 13 and 14 (And Bonus 15 and 16 Updates)

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

Proof of concept pages pushed
Manual tooling now included

### CODE REVIEW

File: tools/user/pgerelnotes/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+1) You will need to obtain copies of the files at
+
+https://github.com/EnterpriseDB/2ndqpostgres/blob/2QREL_13_STABLE_dev/doc/src/sgml/release-2q13.sgml
+
+and
+
+https://github.com/EnterpriseDB/2ndqpostgres/blob/2QREL_14_STABLE_dev/doc/src/sgml/release-2q14.sgml
+
+2) save these files to pge13.sgml and pge14.sgml
```
Comment (gvasquezvargas) [inline tools/user/pgerelnotes/README.md:11]:
```suggestion
2) Replace the existing pge13.sgml and pge14.sgml files under `tools/user/pgerelnotes/` with the updated files you downloaded in the previous step.
```

File: tools/user/pgerelnotes/pgerel/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+1) You will need to obtain copies of the files at
+
+https://github.com/EnterpriseDB/2ndqpostgres/blob/2QREL_13_STABLE_dev/doc/src/sgml/release-2q13.sgml
+
+and
+
+https://github.com/EnterpriseDB/2ndqpostgres/blob/2QREL_14_STABLE_dev/doc/src/sgml/release-2q14.sgml
+
+2) save these files to pge13.sgml and pge14.sgml
```
Comment (gvasquezvargas) [inline tools/user/pgerelnotes/pgerel/README.md:15]:
```suggestion
2) Replace the existing pge13.sgml and pge14.sgml files under `tools/user/pgerelnotes/pgerel` with the updated files you downloaded in the previous step.
```

File: tools/user/pgerelnotes/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+Provisional docs for the XSLT translator
+
```
Comment (gvasquezvargas) [inline tools/user/pgerelnotes/README.md:2]:
```suggestion

**Prerequisite - xsltproc is installed on your system**

```

File: tools/user/pgerelnotes/pgerel/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+3) set $DOCSHOME to $HOME/docs or wherever you keep your docs tree
+
+4) Ensure that your docs repository is on a branch where you wish to update the release notes
+
+5) cd $DOCSHOME/tools/user/pgerelnotes/pgerel
+
+5) Run
```
Comment (gvasquezvargas) [inline tools/user/pgerelnotes/pgerel/README.md:23]:
```suggestion
6) Run
```

File: tools/user/pgerelnotes/pgerel/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+xsltproc --param version 13  transform.xslt pge13wrap.xml >$DOCSHOME/product_docs/docs/pge/13/release_notes/index.mdx
+
+xsltproc --param version 14  transform.xslt pge14wrap.xml >$DOCSHOME/product_docs/docs/pge/14/release_notes/index.mdx
+
+6) Check in the modified files.
```
Comment (gvasquezvargas) [inline tools/user/pgerelnotes/pgerel/README.md:29]:
```suggestion
7) Check in the modified files.
```

File: tools/user/pgerelnotes/README.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+and
+
+https://github.com/EnterpriseDB/2ndqpostgres/blob/2QREL_14_STABLE_dev/doc/src/sgml/release-2q14.sgml
+
+2) save these files to pge13.sgml and pge14.sgml
+
+3) set $DOCSHOME to $HOME/docs or wherever you keep your docs tree
+
+4) Ensure that your docs is checkout into a branch where you wish to update the release notes
+
```
Comment (gvasquezvargas) [inline tools/user/pgerelnotes/README.md:16]:
```suggestion

5) cd $DOCSHOME/tools/user/pgerelnotes/pgerel

```

File: tools/user/pgerelnotes/README.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+5) Run
```
Comment (gvasquezvargas) [inline tools/user/pgerelnotes/README.md:17]:
```suggestion
6) Run
```

File: tools/user/pgerelnotes/README.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+xsltproc --param version 13  transform.xslt pge13wrap.xml >$DOCSHOME/product_docs/docs/pge/13/release_notes/index.mdx
+
+xsltproc --param version 14  transform.xslt pge14wrap.xml >$DOCSHOME/product_docs/docs/pge/14/release_notes/index.mdx
+
+6) Check in the modified files.
```
Comment (gvasquezvargas) [inline tools/user/pgerelnotes/README.md:23]:
```suggestion
7) Check in the modified files.
```

**Review by gvasquezvargas — COMMENTED** [review]
Minor comments on the README files. 
Also, there are 2 README files, do we need both?

**Review by theadamwright — APPROVED** [review]

**Pull request merged**